tests/gpu-tests/run_qwen.sh

```diff
@@ -2,13 +2,16 @@
 # also need to define HF_TOKEN for some of the tests
 set -e
 
-export NEMO_SKILLS_TEST_HF_MODEL=Qwen/Qwen3-4B
 export NEMO_SKILLS_TEST_MODEL_TYPE=qwen
+# TRTLLM still doesn't support Qwen3 models, using a smaller Qwen2.5 model for context retry tests
```

Comment:
it actually does, but you need to use `--backend pytorch` argument in server args

Reply:
oh i see